src/components/AddTable.tsx

```diff
@@ -0,0 +1,42 @@
+import Button from "./Button";
+import { ButtonVarient } from "../types/ButtonTypes";
+
+const AddTable = () => {
```

Comment:
Please rename the component. The component name should specify what the component detail is